dist/leaflet.css

```
@@ -52,6 +52,7 @@
 	max-height: none !important;
 	width: auto;
 	padding: 0;
+	mix-blend-mode: plus-lighter;

```

Perhaps we can add a comment for future reference?
```suggestion
	/* See: https://bugs.chromium.org/p/chromium/issues/detail?id=600120 */
	mix-blend-mode: plus-lighter;

```

---

My location doesn't show actually. How can I fix and update